### ISSUE TITLE
Should meridiem be undefined when implied?

### ISSUE THREAD

**Issue by chadfawcett** [wanasit/chrono#96]
After parsing, you are able to get most implied values. For example:

```javascript
var result = chrono.parse('1-3')
result[0].start.get('month')
//  3  <- Able to get implied month value
```

However when trying to get the implied miridiem:

```javascript
result[0].start.get('meridiem')
//  undefined <- Unable to get implied meridiem
```

Is this the intended behaviour?

**Comment by wanasit** [comment]
That's a difficult edge case. When the time component is not mentioned (eg "2016-03-12"), chrono assume it's as 12.00 (mid-noon) of that day. However, the meridiem component of 12PM is confusing and difficult to get it right.

So we currently don't support getting implied meridiem without mentioning the time.
Do you have a need for that feature?

**Comment by chadfawcett** [comment]
Sorry, my example wasn't very clear. `1-3` was meant to be the time and not a date. So if I change it to:

```javascript
var result = chrono.parse('1-3pm')
```

The same result happens. Basically I just got the `month` value as an example of a property that was implied. 

My specific need is for parsing something like `11pm-2`, I want to check if the end meridiem as AM. Since the meridiem for the end (`2am`) is *implied*, the `get('meridiem')` method returns `undefined`.

As a work around I am currently just checking whether or not the `hour` property is less tan 12. Something like `result.end.get('hour') < 12` which works but isn't as clean.

**Comment by wanasit** [comment]
Yes. The meridiem should be implied when chrono parse time expression.
Fixed in ef76750

**Comment by chadfawcett** [comment]
Great thanks!
